src/main/resources/application.properties

```
@@ -0,0 +1,17 @@
+spring.main.banner-mode=off
+
+spring.jpa.hibernate.ddl-auto=none
+
+spring.sql.init.mode=always
+spring.sql.init.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/test
```

I think test is a bad hardcoded datatbase name. We should rather use `crosswordpuzzle` or somthing like that
```suggestion
spring.datasource.url=jdbc:postgresql://localhost:5432/crosswordpuzzle
```